### ISSUE TITLE
Mouse movement is not even close to 1:1 with the host and wonky/console port-y, or just not good.

### ISSUE THREAD

**Issue by thiccelf83** [LizardByte/Sunshine#167]
### Describe the Bug

On the host system (windows) mouse movement is within the parameters of the settings for it. When streaming a computer, windows or linux, mouse movement is far from the same. even if the settings and os on the guest and hoist are matching. I have experienced this with geforce experience too and it seems to be an inherit issue of the software, or maybe just moonlight.

It's unusable in some shooters, those with inherit input lag issues especially such as the early battlefield series. but just in general, I can never perform when streaming (locally always) as I would on my pc, because the mouse movement is off. Its accelerated and weird, its like how console port aiming is at times.

This is probably a issue with the client and not the streaming software, but I am not a developer so i wouldnt know.

One more thing is, it seems that this is somewhat different every session.

### Expected Behavior

mouse movement should be more or less 1:1 with the host system within reasonableamounts and not strangely accelerated in some weird emulated hard to predict way. 

### Additional Context

I've been told before this is unfixable and an inherit problem with gamestream.

 but this is sunshine. 

### Sunshine Host Operating System and Version

Windows 10

### Architecture

64 bit

### Sunshine Version

0.13.0

### GPU Type

nvidia

### GPU Model

gtx 980 ti

### GPU Driver/Mesa Version

27.21.14.5671

**Comment by cgutman** [comment]
With GFE, the problem is that it turns on mouse acceleration and adjusts your mouse pointer speed when streaming starts. That’s the “unfixable” issue. 

Sunshine doesn’t do that, but there have historically been other mouse issues on systems such multiple monitors. I’m not sure if any of those still exist these days.

Are you using multiple monitors on your host?

What client OS are you running Moonlight on? I assume you have “remote desktop optimized mouse mouse” off in Moonlight, correct?

**Comment by thiccelf83** [comment]
Hi, thanks for replying. I am not using GFE anymore, just sunshine. 

Yes, that is off. Its less wonky, more that it is accelerated and weird and not 1:1, if I matc hsettings, its a little better. but its not like native, and the difference is too great to be chalked up to latency on a local network like mine where performance is fine.

It feels emulated

**Comment by thiccelf83** [comment]
Is this related to my cursor not showing without trails on any version of sunshine

**Comment by wezel** [comment]
Hi, I've encountered a similar issue on a Linux (pop-os 22.04) host.
Except for me the behavior is somewhat predictable: The cursor always prefers to go to the top left corner.

I ran some tests by moving the cursor in specific ways:
- circles -> ends up in the top left corner.
- horizontal lines -> ends up in the top left corner.
- vertical lines -> ends up in the top.
- skew lines bottom left to top right ->
         With mouse acceleration on the client cursor sometimes only moves horizontally.
         Without mouse acceleration on the client it drifts to the top left.

if you want I can create a separate bug report and or provide you with more information about my specific setup.
